tlf.1.in

```diff
@@ -152,6 +153,18 @@ Options given to @PACKAGE_NAME@ on the command line.
 Print a summary of options to the screen and exit.
 .
 .TP
+.BI \-d\  debug_level
+.TQ
+.BI \-\-debug=  debug_level
+Activate logging of debug output. Output will be added to
+.I debug.txt
+file in actual directory.
+.
+.IP
+Allowed values with rising verbosity are 0 (Off), 1 (Error) , 2 (Warn), 3
```

Comment:
just a thought: do we need 0 (Off) here? it's off by default, so there is no point specifying -d0

Reply:
You are totally right. it makes no sense. See coming commit